.github/workflows/e2e_ios.yml

```diff
@@ -7,7 +7,7 @@ name: '[e2e] iOS Detox Test Run'
 on:
   push:
     branches:
-      - 'dependabot/**'
+      - develop
```

Comment:
is the objective to run the detox tests after it is pushed to develop? That is what this line would do